pkg/models/apikey.go

```diff
@@ -7,6 +7,7 @@ import (
 
 var ErrInvalidApiKey = errors.New("Invalid API Key")
 var ErrInvalidApiKeyExpiration = errors.New("Negative value for SecondsToLive")
+var ErrDuplicateApiKey = errors.New("UNIQUE constraint failed: api_key.org_id, api_key.name")
```

Comment:
Finally, this can become:
`var ErrDuplicateApiKey = errors.New("API Organisation id and Key Name Must Be Unique")`